src/main/java/bio/terra/common/logging/GoogleJsonLayout.java

```diff
@@ -55,6 +55,7 @@
  * href="https://github.com/ankurcha/gcloud-logging-slf4j-logback/">gcloud-logging-slf4j-logback</a>
  * which inspired some of the patterns used here.
  */
+@SuppressWarnings("MultipleStringLiterals")
```

Comment:
ditto I don't think good style is automatable and adding suppression warnings feels silly.

Reply:
for spotbugs at least, the new annotation is `@SuppressFBWarnings` http://findbugs.sourceforge.net/api/edu/umd/cs/findbugs/annotations/SuppressFBWarnings.html